src/lists/list-item.jsx

```diff
@@ -198,10 +198,6 @@ const ListItem = React.createClass({
         left: 16,
       },
 
-      primaryText: {
```

Comment:
I feel that in the future if we do decide to add some styling for the `primaryText`, leaving it like that would make it easier to locate, and it also feels more consistent with all the other style key:value pairs.

It's up to you though. Let me know what you decide. :-)